packages/forms/src/UIForm/fields/Text/Text.component.js

```diff
@@ -48,7 +47,22 @@ export default function Text(props) {
 		step: get(schema, 'schema.step'),
 		...extractDataAttributes(rest),
 	};
-
+	if (type === 'password') {
+		return (
+			<Form.Password
+				{...fieldProps}
+				id={id}
+				label={title}
+				required={schema.required}
+				hasError={!!errorMessage}
+				description={description}
+				aria-invalid={!isValid}
+				aria-required={get(schema, 'required')}
+				aria-describedby={`${descriptionId} ${errorId}`}
+				link={link && <Link {...omit(link, ['label'])}> {link.label} </Link>}
```

Comment:
Not sure about this part as the link prop will create a Link component in the design-system 🤔 
https://github.com/Talend/ui/blob/master/packages/design-system/src/components/Form/Primitives/Field/Field.tsx#L73

Reply:
It is on master I put it to not change the behavior. I don't know :shrug:  what is the use case for it.